src/ConsoleTables/ConsoleTable.cs

```diff
@@ -219,20 +254,30 @@ public void Write(Format format = ConsoleTables.Format.Default)
         }
 
         private static IEnumerable<string> GetColumns<T>()
-        {  
+        {
             return typeof(T).GetProperties().Select(x => x.Name).ToArray();
         }
 
         private static object GetColumnValue<T>(object target, string column)
         {
             return typeof(T).GetProperty(column).GetValue(target, null);
         }
+
+        private static IEnumerable<Type> GetColumnsType<T>()
+        {
+            return typeof(T).GetProperties().Select(x => x.PropertyType).ToArray();
+        }
     }
 
     public class ConsoleTableOptions
     {
         public IEnumerable<string> Columns { get; set; } = new List<string>();
         public bool EnableCount { get; set; } = true;
+
+        /// <summary>
+        /// Enable only from a list of objects
+        /// </summary>
+        public bool NumberRigthAligned { get; set; }
```

Comment:
Would this be better as an enum value? `Left`, `Right`, `Center` for example.

Reply:
I thought about it, but string.format does not have center alignment